Resources/Prototypes/Civ14/Entities/Objects/Weapons/spears.yml

```
@@ -92,11 +102,23 @@
     - type: Sprite
       sprite: Civ14/Weapons/naginata.rsi
     - type: MeleeWeapon
+      range: 1.6
       damage:
         types:
-          Slash: 47
+          Slash: 1
+          Piercing: 13
+    - type: IncreaseDamageOnWield
+      damage: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)